packages/services/usage/src/usage.ts

```diff
@@ -32,7 +41,7 @@ const levelMap = {
 
 const retryOptions = {
   maxRetryTime: 30_000,
-  initialRetryTime: 500,
+  initialRetryTime: 300,
```

Comment:
A continuation of more aggressive times in general. 300ms is the default for kafkajs and I don't see a reason to wait longer.